dbt/models/ferc1/out_ferc1__yearly_steam_plants_fuel_by_plant_sched402/schema.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+sources:
+  - name: pudl
+    tables:
+      - name: out_ferc1__yearly_steam_plants_fuel_by_plant_sched402
+        data_tests:
+          - check_row_counts_per_partition:
+              table_name: out_ferc1__yearly_steam_plants_fuel_by_plant_sched402
+              partition_column: report_year
+        columns:
+          - name: report_year
+          - name: utility_id_ferc1
+          - name: utility_id_pudl
+          - name: utility_name_ferc1
+          - name: plant_id_pudl
+          - name: plant_name_ferc1
+          - name: coal_fraction_cost
+          - name: coal_fraction_mmbtu
+          - name: fuel_cost
+          - name: fuel_mmbtu
+          - name: gas_fraction_cost
+          - name: gas_fraction_mmbtu
+          - name: nuclear_fraction_cost
+          - name: nuclear_fraction_mmbtu
+          - name: oil_fraction_cost
+          - name: oil_fraction_mmbtu
+          - name: primary_fuel_by_cost
+          - name: primary_fuel_by_mmbtu
+          - name: waste_fraction_cost
+          - name: waste_fraction_mmbtu
+
+models:
+  - name: validate_ferc1__yearly_steam_plants_fuel_by_plant_sched402
```

Comment:
In this case where we've created an ephemeral derived table (model) to run a test against, this information is all compiled by hand, right?  And `dbt` just knows to automatically look for a `.sql` file with the same name as the model to use that to create the table? Or the `.sql` is picked up by `dbt build` and that instantiates the model I guess.